recipes/TXM_Sandbox/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Could you please fix next release so that the LICENSE is part of the sdist?
You can add a `MANIFEST.in`. See https://packaging.python.org/en/latest/guides/using-manifest-in/

Reply:
LICENSE is added in sdist.

Reply:
Nice. Then you don't need that file in the recipe anymore. Please remove it.